       1:      1  1 CALL pred ho_5.main/2-0 (cc_multi) ho_5.m:8
mdb> echo on
Command echo enabled.
mdb> register --quiet
mdb> break p
 0: + stop  interface pred ho_5.p/2-0 (det)
mdb> continue
      12:      9  7 CALL pred ho_5.p/2-0 (det) ho_5.m:18 (exception.m:NNNN)
mdb> finish
      25:      9  7 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:18 (exception.m:NNNN)
mdb> dd -d 3 -n 7
Call p(1, _)
Throws zero
Expected? no
q(1, 0)
Valid? yes
Call r(0, _)
Throws zero
Expected? yes
Found unhandled or incorrect exception:
p(1, _)
zero
Is this a bug? yes
      25:      9  7 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:18 (exception.m:NNNN)
mdb> continue
exception(univ_cons('<<function>>'))
    1603:    608  7 CALL pred ho_5.p/2-0 (det) ho_5.m:18 (exception.m:NNNN)
mdb> finish
    1616:    608  7 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:18 (exception.m:NNNN)
mdb> dd -d 3 -n 7
Call p(2, _)
Throws zero
Expected? no
q(2, 0)
Valid? yes
Found unhandled or incorrect exception:
p(2, _)
zero
Is this a bug? yes
    1616:    608  7 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:18 (exception.m:NNNN)
mdb> continue
exception(univ_cons('<<function>>'))
